ui.R

```diff
@@ -14,7 +14,8 @@ dashboardPage(
       tags$meta(name="keywords", content="MetaInsight, NMA, Network, Meta, Analysis, App"),
       tags$meta(property="og:title", content="Meta Insight: v5.0.1"),
       tags$meta(property="og:description", content="An interactive web tool for network meta-analysis (NMA) that leverages established analysis routines"),
-      tags$meta(property="og:image", content="https://raw.githubusercontent.com/CRSU-Apps/MetaInsight/main/www/images/MetaInsightLogo.png")
+      tags$meta(property="og:image", content="https://raw.githubusercontent.com/CRSU-Apps/MetaInsight/main/www/images/MetaInsightLogo.png"),
+      google_analytics_header_ui(id = "analytics")
     ),
     navbarPage(
       title = "MetaInsight", 
```

Comment:
The line below (`header...`) needs to be removed otherwise google analytics is loaded all the time.

Reply:
Removed.